templates/web/base/report/new/form_report.html

```diff
@@ -49,13 +49,22 @@ <h2 class="form-section-heading">[% loc( 'Public details' ) %]</h2>
 [% TRY %][% PROCESS 'report/new/after_photo.html' %][% CATCH file %][% END %]
 
     [% DEFAULT form_detail_label = loc('Explain what’s wrong') %]
-    [% DEFAULT form_detail_placeholder = loc('e.g. ‘This pothole has been here for two months and…’') %]
```

Comment:
What you have works fine, but I wonder if it would be clearer if this line became `[% IF NOT  form_detail_placeholder.defined %][% SET form_detail_placeholder = ... %][% END %]` and then below would be `[% IF form_detail_placeholder %]...` without needing to duplicate e.g. the aria_describedby line. What do you think? Not much in it.